docker-compose.yml

```diff
@@ -1,6 +1,8 @@
 version: "2"
 services:
   app:
+    env_file: 
+    - .env
```

Comment:
This pulls the .env file so that the app can be ran in Docker, otherwise the app complains about missing some keys - this was added during testing, and isn't really part of this PR
but if this is ok then update README.md `docker-compose up` command to `docker compose up`